Content.Server/Atmos/Piping/Unary/EntitySystems/GasVentScrubberSystem.cs

```diff
@@ -59,40 +56,35 @@ private void OnVentScrubberUpdated(EntityUid uid, GasVentScrubberComponent scrub
             || !EntityManager.TryGetComponent(uid, out NodeContainerComponent? nodeContainer)
             || !nodeContainer.TryGetNode(scrubber.OutletName, out PipeNode? outlet))
             {
-                appearance?.SetData(ScrubberVisuals.State, ScrubberState.Off);
-                _ambientSoundSystem.SetAmbience(scrubber.Owner, false);
                 return;
             }
-            _ambientSoundSystem.SetAmbience(scrubber.Owner, true);
 
-            var environment = _atmosphereSystem.GetTileMixture(EntityManager.GetComponent<TransformComponent>(scrubber.Owner).Coordinates, true);
+            var xform = Transform(uid);
+            var environment = _atmosphereSystem.GetTileMixture(xform.Coordinates, true);
 
-            Scrub(timeDelta, scrubber, appearance, environment, outlet);
+            Scrub(timeDelta, scrubber, environment, outlet);
 
             if (!scrubber.WideNet) return;
 
             // Scrub adjacent tiles too.
-            foreach (var adjacent in _atmosphereSystem.GetAdjacentTileMixtures(EntityManager.GetComponent<TransformComponent>(scrubber.Owner).Coordinates, false, true))
+            foreach (var adjacent in _atmosphereSystem.GetAdjacentTileMixtures(xform.Coordinates, false, true))
```

Comment:
This `GetAdjacentTileMixtures` override apparently doesn't pass along the `includeBlocked` and `invalidate` arguments, I guess it might as well just get fixed in this PR.